bin/custom/questions/acceleratoraddon/web/src/com/expertsoft/questions/controllers/cms/QuestionsCMSComponentController.java

```diff
@@ -0,0 +1,41 @@
+package com.expertsoft.questions.controllers.cms;
+
+import com.expertsoft.model.cms2.components.QuestionsCMSComponentModel;
+import com.expertsoft.questions.model.QuestionModel;
+import de.hybris.platform.addonsupport.controllers.cms.AbstractCMSAddOnComponentController;
+import de.hybris.platform.converters.impl.AbstractConverter;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.RequestMapping;
+import questions.data.QuestionData;
+
+import javax.annotation.Resource;
+import javax.servlet.http.HttpServletRequest;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+@Controller("QuestionsCMSComponentController")
+@RequestMapping(value = "/view/QuestionsCMSComponentController")
```

Comment:
It's a good practice to use constants like _Prefix + QuestionsCMSComponentModel._TYPECODE + _Suffix